api/src/main/java/org/openmrs/module/openconceptlab/Subscription.java

```diff
@@ -28,10 +28,18 @@
 
 	private Integer minutes = 0;
 
-	private boolean subscribedToSnapshot = true;
+	private Boolean subscribedToSnapshot = false;
 
-	public Subscription() {
+	private Boolean validation = true;
 
+	public Subscription() {}
+
+	public Boolean isValidation() {
```

Comment:
Yeah I did the enum first but then i noticed that you wanted a checkbox(validate or not) so I changed it to boolean.